src/types.ts

```diff
@@ -41,12 +41,18 @@ export interface TokenInfoJSON {
   pegTo?: string
   isSupercharged?: boolean
   bridge?: string
+  isStableCoin?: boolean
+  isZeroState?: boolean
```

Comment:
Why do we need this? And would there be a more descriptive name?
```suggestion
  showZeroBalance?: boolean
```

Reply:
This is for showing the token even if the balance is zero in the new [Assets screen](https://www.figma.com/file/OE6RFzqXAMqU9ygHtNYyof/Assets-Page-(Multichain)?node-id=2099%3A17616&mode=dev). And I've updated the name